apps/widget-configurator/src/app/configurator/index.tsx

```diff
@@ -48,16 +57,43 @@ export function Configurator({ title }: { title: string }) {
   const [buyToken, setBuyToken] = buyTokenState
   const [buyTokenAmount, setBuyTokenAmount] = buyTokenAmountState
 
-  const iframeUrl = useMemo(() => {
-    return buildIframeUrl({
-      chainId,
-      sellTokenId: sellToken,
-      sellAmount: sellTokenAmount,
-      buyTokenId: buyToken,
-      buyAmount: buyTokenAmount,
-      theme: mode,
-    })
-  }, [chainId, sellToken, sellTokenAmount, buyToken, buyTokenAmount, mode])
+  const iframeContainerRef = useRef<HTMLDivElement>(null)
+  const updateWidgetRef = useRef<UpdateWidgetCallback | null>(null)
+
+  useEffect(() => {
+    const widgetContainer = iframeContainerRef.current
+
+    if (!widgetContainer) return
+
+    const params: CowSwapWidgetParams = {
+      container: widgetContainer,
+      metaData: { appKey: 'YOUR_APP_ID', url: 'https://YOUR_APP_URL' },
+      width: 400,
```

Comment:
@fairlighteth would it be good thing to allow to change this? 

I think, at least 2 options:
- Widget mode: `400`
- Responsive: `100%`